themes/sdss_subtheme/templates/components/news-vertical-teaser/pattern-news-vertical-teaser.ui_patterns.yml

```diff
@@ -32,6 +37,11 @@ sdss-news-vertical-teaser:
       label: "URL for item"
       description: "URL for image and heading"
       preview: "http://www.stanford.edu/"
+    su_news_source:
+      type: text
+      label: "External URL for item"
+      description: "URL for image and heading"
+      preview: "http://www.google.com/"
```

Comment:
Same question if we'd want to rename `su_news_source` to `news_external_url`

Reply:
Ok, all done.